lib/capistrano/hivequeen/capistrano_configuration.rb

```diff
@@ -88,19 +88,39 @@
         logger.trace "Using #{db_server} as primary db server"
         role :db, db_server.to_s, :primary => true
       end
-
     end
   end
 
   namespace :ssh do
+    command_format = "ssh -t -A -l %s %s"
     HiveQueen.default_roles.each do |role_name|
       task role_name do
-        cmd =  "ssh -t -A -l #{user} #{roles[role_name.to_sym].servers.first}"
+        cmd = command_format % [user, roles[role_name.to_sym].servers.first]
         puts "Executing #{cmd}"
         exec cmd
       end
     end
 
+    task :default do
+      cmd = command_format % [user, roles.values.sample.servers.sample]
+      puts "Executing #{cmd}"
+      exec cmd
+    end
+  end
+
+  namespace :console do
+    command_format = "ssh -t -A -l %s %s 'source /etc/profile; cd /apps/#{HiveQueen.project}/current && bundle exec rails console'"
+    HiveQueen.default_roles.each do |role_name|
+      task role_name do
+        puts "Opening console"
+        exec command_format % [user, roles[role_name.to_sym].servers.first]
+      end
+    end
+
+    task :default do
+      puts "Opening console"
+      exec command_format % [user, roles.values.sample.servers.sample]
```

Comment:
How about using `bg` role by default? Wouldn't want a resource-intensive ruby console to affect web/app roles.